src/utilities/color/color.ts

```diff
@@ -29,6 +29,19 @@ export const SWATCH_COLORS = [
 ];
 export const DEFAULT_COLOR = SWATCH_COLORS[0];
 
+// Colors for AST image overlay
+export const AST_COLORS = new Map<number, string>([
+    [0, "auto-black"],
+    [1, "auto-white"],
+    [2, "auto-red"],
+    [3, "auto-forest"],
+    [4, "auto-blue"],
+    [5, "auto-turquoise"],
+    [6, "auto-violet"],
+    [7, "auto-gold"],
+    [8, "auto-gray"]
+]);
+
```

Comment:
1. This can just be an array, since the "keys" are sequential integers starting from zero.
2. Since this is *only* used for conversion of a legacy preferences format, it may be better to define this directly in the upgrade function.